rs/consensus/src/idkg/payload_builder/pre_signatures.rs

```diff
@@ -365,7 +364,7 @@ fn make_new_pre_signatures_if_needed_helper(
                 MasterPublicKeyId::Ecdsa(_) | MasterPublicKeyId::Schnorr(_)
             )
         })
-        .find(|key_config| &key_config.key_id == key_id)
+        .find(|key_config| &key_config.key_id == key_id.inner())
```

Comment:
Do we still need the filter above, now that the function is guaranteed to be called with an `IDkgMasterPublicKeyId`?

Reply:
Good catch! No we don't

Reply:
Removed it